desktop/apps/consign/components/app/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import ResponsiveWindow from 'desktop/components/react/responsive_window'
+import SubmissionFlow from 'desktop/apps/consign/components/submission_flow'
+import { Provider } from 'react-redux'
+import { Redirect, Route, Switch } from 'react-router'
+import { data as sd } from 'sharify'
```

Comment:
no, it's supposed to run on both. Ok noted, I'll pass in a prop with user data then.